components/datatypes/data/order.schema.json

```diff
@@ -64,6 +71,46 @@
           },
           "meta:titleId": "order##xdm:orderType##title##56861",
           "meta:descriptionId": "order##xdm:orderType##description##62731"
+        },
+        "xdm:lastUpdatedDate": {
+          "title": "Last Updated Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "The time when a particular order record is last updated in the commerce system.",
+          "meta:titleId": "order##xdm:lastUpdatedDate##title##86152",
+          "meta:descriptionId": "order##xdm:lastUpdatedDate##description##47783"
+        },
+        "xdm:createdDate": {
+          "title": "Created Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "The time/date when a new order is created in the commerce system.",
+          "meta:titleId": "order##xdm:createdDate##title##86153",
+          "meta:descriptionId": "order##xdm:createdDate##description##47783"
+        },
+        "xdm:shipDate": {
```

Comment:
Please refer the "shipping" property in the field group https://github.com/adobe/xdm/blob/master/components/datatypes/marketing/commerce.schema.json

This property refers a "shipping" data type ( https://github.com/adobe/xdm/blob/master/components/datatypes/shipping.schema.json ) which has some shipping related information. Please add properties to this data type if there is a need to enhance this or add more shipping related properties for an order.